packages/connect-core/src/utils/description.ts

```diff
@@ -0,0 +1,94 @@
+import { ethers } from 'ethers'
```

Comment:
Here is the main logic we are currently using to both encode and decode an evm script and transform it into a `ForwardingPathDescrption`. I'm not convinced yet about the current data types and how we constructing the description but I think we are going in the right direction. Maybe it's a big topic and we should address it as a separate PR and further discuss it in: https://github.com/aragon/connect/issues/219